From 5c50f5abad98949a834c0a2ebf71dc581a3360e2 Mon Sep 17 00:00:00 2001
From: Jagdish Gediya <jagdish.gediya@nxp.com>
Date: Wed, 15 Aug 2018 22:32:01 +0530
Subject: [PATCH 088/107] armv8: layerscape: fix compilation warning 'unused ns_dev'

unnecessary include of 'ns_access.h' file is causing compilation
warning because 'ns_dev' is defined static in header file but
it is not used in source file.

compilation is tested for la1012a*, ls1043a* and ls1046a* boards.

Signed-off-by: Jagdish Gediya <jagdish.gediya@nxp.com>
---
 arch/arm/cpu/armv8/fsl-layerscape/soc.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/arm/cpu/armv8/fsl-layerscape/soc.c b/arch/arm/cpu/armv8/fsl-layerscape/soc.c
index 5f00d3c..0496cf0 100644
--- a/arch/arm/cpu/armv8/fsl-layerscape/soc.c
+++ b/arch/arm/cpu/armv8/fsl-layerscape/soc.c
@@ -14,7 +14,6 @@
 #include <asm/io.h>
 #include <asm/global_data.h>
 #include <asm/arch-fsl-layerscape/config.h>
-#include <asm/arch-fsl-layerscape/ns_access.h>
 #include <asm/arch-fsl-layerscape/fsl_icid.h>
 #ifdef CONFIG_LAYERSCAPE_NS_ACCESS
 #include <fsl_csu.h>
-- 
1.7.1

